<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDataPopup</class>
 <widget class="QDialog" name="GameDataPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="awayTeamLayout">
     <item>
      <widget class="QLabel" name="aTeamLabel">
       <property name="text">
        <string>Away Team</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="aTeamLabelView"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="homeTeamLayout">
     <item>
      <widget class="QLabel" name="hTeamLabel">
       <property name="text">
        <string>Home Team</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="hTeamTableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
